.gitattributes

```diff
@@ -0,0 +1,4 @@
+/.yarn/**            linguist-vendored
```

Comment:
why check these in? shouldn't we just git ignore `.yarn`?

Reply:
that's the file generated by yarn init. I think it's safe to keep it as is

Reply:
yarn modern is super opinionated about determinism in a way that I'm not sure aligns with our needs here. I'd much prefer to not check in `.yarn/` and rely on `yarn.lock` being sufficient